examples/event_listener/event_listener.py

```diff
@@ -0,0 +1,59 @@
+import json
+import os
+import signal
+from os import listdir
+from os.path import join, isfile
+from pathlib import Path
+
+from astarte.device import Device
+
+_ROOT_DIR = Path(__file__).parent.absolute()
+_INTERFACES_DIR = os.path.join(Path(__file__).parent.absolute(), 'interfaces')
+_DEVICE_ID = 'DEVICE_ID_HERE'
+_REALM = 'REALM_HERE'
+_CREDENTIAL_SECRET = 'CREDENTIAL_SECRET_HERE'
+_PAIRING_URL = 'https://PAIRING_URL_HERE'
+
+
+class ProgramKilled(Exception):
+    pass
+
+
+def signal_handler(signum, frame):
+    print("Shutting Down...")
+    raise ProgramKilled
+
+
+def _load_interfaces() -> [dict]:
+    files = [join(_INTERFACES_DIR, f) for f in listdir(_INTERFACES_DIR) if isfile(join(_INTERFACES_DIR, f))]
+    interfaces = []
+    for file in files:
+        with open(file, 'r') as interface_file:
+            interfaces.append(json.load(interface_file))
+    return interfaces
+
+
+def callback(device: Device, name: str, path: str, payload: dict):
+    print(f"Received message for {name}{path}: {payload}")
+
+
+def main():
+    device_id = _DEVICE_ID
+    realm = _REALM
+    credentials_secret = _CREDENTIAL_SECRET
+    pairing_url = _PAIRING_URL
```

Comment:
Just pass the original variables to initialize the device, these assignments feel superfluous and do not add value or readability to the example